press-kit/index.md

```diff
@@ -120,7 +120,7 @@ meta_descr: meta_descr.presskit
   <div class="info-block">
     <h2>{% t press-kit.pressdoc %}</h2>
     <div>
-        <h3><a href="https://www.monerooutreach.org/quick-facts.html">{% t press-kit.quickfacts %}</a></h3>
+        <h3><a href="https://resilience365.com/monero-quick-facts">{% t press-kit.quickfacts %}</a></h3>

```

Comment:
```suggestion
        <h3><a href="https://resilience365.com/files/QuickFacts.pdf">{% t press-kit.quickfacts %} (PDF)</a></h3>
```
i think this is what @HardenedSteel suggests. 
Linking to our hosted version would be ideal though.

I'm ok with any of them.
A) linking to html
B) linking to external PDF
C) linking to internal PDF